### PR TITLE
Fix nodenames in the docs

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Fix nodenames in the docs.
  - A few missing nodes are missing in the list. Might want to add those in the future.

### CODE REVIEW

**Comment by SmallJoker** [comment]
I think the aliases caused some confusion because they follow a different naming scheme. Would you please be so nice to add a link to those? https://github.com/minetest-mods/maptools/blob/master/aliases.lua#L8

**Comment by Panquesito7** [comment]
> I think the aliases caused some confusion because they follow a different naming scheme. Would you please be so nice to add a link to those? https://github.com/minetest-mods/maptools/blob/master/aliases.lua#L8

Sure, sorry for the very late reply.
Should I add a link to the aliases in the Markdown file or?

**Comment by SmallJoker** [comment]
The alias code is very easy to read, hence I think that linking to the source file directly is enough to document the available aliases. No need to bloat the documentation with that.

File: docs/NODES_ITEMS.md [hunk 1]
```diff
@@ -12,15 +12,19 @@ use `-1` as the amount.
...
 | `maptools:{block}_u`    | Unbreakable, non-flammable, non-falling, non-decaying blocks. Most common default blocks have an unbreakable form available (`maptools:stone_u`, `maptools:wood_u`, …) |
 | `maptools:full_grass`   | Unbreakable block with the grass texture on all sides.                                                                                                                 |
-| `maptools:player_clip`  | :warning: Invisible, non-pointable block that blocks players and entities.                                                                                             |
-| `maptools:full_clip`    | Invisible, pointable block that blocks players and entities. Also available as a thin face (`maptools:full_clip_face`).                                                |
+| `maptools:playerclip`  | :warning: Invisible, non-pointable block that blocks players and entities.                                                                                             |
+| `maptools:fullclip`    | Invisible, pointable block that blocks players and entities. Also available as a thin face (`maptools:fullclip_face`).                                                |
```
Comment (Calinou) [inline docs/NODES_ITEMS.md]:
```suggestion
| `maptools:playerclip`  | :warning: Invisible, non-pointable block that blocks players and entities.                                                                                              |
| `maptools:fullclip`    | Invisible, pointable block that blocks players and entities. Also available as a thin face (`maptools:fullclip_face`).                                                  |
```

File: docs/NODES_ITEMS.md [hunk 2]
```diff
@@ -12,15 +12,19 @@ use `-1` as the amount.
...
 | `maptools:smoke`        | Some smoke. Decreases visibility, but doesn't damage players or entities).                                                                                             |
-| `maptools:no_build`     | :warning: Very basic building prevention.                                                                                                                              |
-| `maptools:no_interact`  | Prevents interacting through the block (opening chests, furnaces, attacking entities, …), but can still be walked through.                                             |
+| `maptools:nobuild`     | :warning: Very basic building prevention.                                                                                                                              |
+| `maptools:nointeract`  | Prevents interacting through the block (opening chests, furnaces, attacking entities, …), but can still be walked through.                                             |
```
Comment (Calinou) [inline docs/NODES_ITEMS.md]:
```suggestion
| `maptools:nobuild`     | :warning: Very basic building prevention.                                                                                                                               |
| `maptools:nointeract`  | Prevents interacting through the block (opening chests, furnaces, attacking entities, …), but can still be walked through.                                              |
```

File: docs/NODES_ITEMS.md [hunk 3]
```diff
@@ -31,10 +35,10 @@ as they cannot be removed by hand (they can only be removed with
...
-| `maptools:infinite_fuel`         | Fuel lasting for a near-infinite time (about 50 real-life years).                                                                                 |
-| `maptools:super_apple`           | A yellow apple which heals the player by 20 HP when used.                                                                                         |
+| `maptools:pick_admin`            | A bright magenta pickaxe with infinite durability, digs everything including unbreakable blocks instantly. No drops are given when digging nodes. |
+| `maptools:pick_admin_with_drops` | Same as the admin pickaxe, but drops are given when digging nodes.                                                                                |
+| `maptools:infinitefuel`         | Fuel lasting for a near-infinite time (about 50 real-life years).                                                                                 |
+| `maptools:superapple`           | A yellow apple which heals the player by 20 HP when used.                                                                                         |
```
Comment (Calinou) [inline docs/NODES_ITEMS.md]:
```suggestion
| `maptools:infinitefuel`         | Fuel lasting for a near-infinite time (about 50 real-life years).                                                                                  |
| `maptools:superapple`           | A yellow apple which heals the player by 20 HP when used.                                                                                          |
```

File: docs/NODES_ITEMS.md [hunk 4]
```diff
@@ -12,15 +12,19 @@ use `-1` as the amount.
 as they cannot be removed by hand (they can only be removed with
 [WorldEdit](https://github.com/Uberi/Minetest-WorldEdit) or similar).
 
+> **Note**
+> All of the nodes mentioned below have aliases which can be used as well.\
+> Here's the full aliases list: <https://github.com/minetest-mods/maptools/blob/master/aliases.lua>
```
Comment (Calinou) [inline docs/NODES_ITEMS.md]:
```suggestion
> **Note**
>
> All of the nodes mentioned below have aliases which can be used as well.\
> Here's the full aliases list: <https://github.com/minetest-mods/maptools/blob/master/aliases.lua>
```

**Pull request merged**